jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-metadata.js

```diff
@@ -17,160 +17,149 @@ define([
 ) {
     'use strict';
 
-    var CELL_PROPERTIES = ['col','row','width','height'];
+    var SPEC_VERSION = 1;
+    var GRID_DEFAULT = 'grid_default';
+    var REPORT_DEFAULT = 'report_default';
 
-    // must match `dashboard-actions` auth states
-    var DASHBOARD_LAYOUT = Object.freeze({
+    // must match `dashboard-actions.js` auth states
+    var DASHBOARD_VIEW = Object.freeze({
         GRID: 'grid',
         REPORT: 'report'
     });
 
-    // pull cell metadata from element data
-    function _getCellMetadata($elem) {
+    // Mapping of view type to view id
+    // This will change when multiple views of the same type are supported.
+    var VIEW_TO_ID = {
+        grid: GRID_DEFAULT,
+        report: REPORT_DEFAULT
+    };
+
+    function _getActiveView() {
+        var metadata = _getDashboardMetadata();
+        if (metadata) {
+            return metadata.activeView;
+        }
+    }
+
+    // get cell view metadata from element data
+    function _getCellViewMetadata($elem, viewId) {
```

Comment:
I see it now. Maybe add some doc on it then indicating the default behavior of the param (2nd right?) is missing.